sections/video.liquid

```diff
@@ -101,7 +101,7 @@
         "youtube",
         "vimeo"
       ],
-      "default": "https:\/\/www.youtube.com\/watch?v=_9VUPq3SxOc",
+      "default": "https://www.youtube.com/watch?v=_9VUPq3SxOc",
```

Comment:
I don't believe those are necessary. You can parse the forward slashes just fine?

```
[25] pry(main)> JSON.parse('{ "url": "https:\/\/www.youtube.com\/watch?v=_9VUPq3SxOc" }')
=> {"url"=>"https://www.youtube.com/watch?v=_9VUPq3SxOc"}
[26] pry(main)> JSON.parse('{ "url": "https://www.youtube.com/watch?v=_9VUPq3SxOc" }')
=> {"url"=>"https://www.youtube.com/watch?v=_9VUPq3SxOc"}
```